docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,234 @@
+# Preparing the LFRQA Dataset
+
+## Overview
+
+The **LFRQA dataset** was introduced in the paper [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998). It features **1,404 science questions** (along with other categories) that have been human-annotated with answers. This tutorial walks through the process of setting up the dataset for use.
+
+## Step 1: Download the Annotations
+
+First, we need to obtain the annotated dataset from the official repository:
+
+```bash
+# Clone the RAG-QA Arena repository
+!git clone <https://github.com/awslabs/rag-qa-arena>
+
+# Create a new directory for the dataset
+!mkdir rag-qa-benchmarking
+
+# Move the science annotations file to our working directory
+!mv rag-qa-arena/data/annotations_science_with_citation.jsonl ./rag-qa-benchmarking/
+
+# Clean up the repository folder
+!rm -rf rag-qa-arena
+```
+
+## Step 2: Download the Robust-QA Documents
+
+LFRQA is built upon **Robust-QA**, so we must download the relevant documents:
+
+```bash
+# Download the Lotte dataset, which includes the required documents
+!curl <https://downloads.cs.stanford.edu/nlp/data/colbert/colbertv2/lotte.tar.gz> --output lotte.tar.gz
+
+# Extract the dataset
+!tar -xvzf lotte.tar.gz
+
+# Move the science test collection to our dataset folder
+!cp lotte/science/test/collection.tsv ./rag-qa-benchmarking/science_test_collection.tsv
+
+# Clean up unnecessary files
+!rm lotte.tar.gz
+!rm -rf lotte
+```
+
+For more details, refer to the original paper: [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998).
+
+## Step 3: Load the Data
+
+We now load both the questions and documents into a usable format:
+
+```python
+import os
+import pandas as pd
+
+# Load questions and answers dataset
+questions = pd.read_json(
+    "rag-qa-benchmarking/annotations_science_with_citation.jsonl", lines=True
+)
+
+# Load documents dataset
+docs = pd.read_csv(
+    "rag-qa-benchmarking/science_test_collection.tsv",
+    sep="\\t",
```

Comment:
Is it really `\\t`? I think it's `\t`, and looking at https://stackoverflow.com/questions/9652832/how-to-load-a-tsv-file-into-a-pandas-dataframe it's `\t` there too